docs_src/hardwaresetup.md

```diff
@@ -10,6 +10,23 @@ Ubuntu [20.04](https://releases.ubuntu.com/focal/) following these [steps](https
 
 Install Docker Engine using these [steps](https://docs.docker.com/engine/install/ubuntu/)
 
+To avoid typing sudo whenever running the docker command, add your username to the docker group:
+
+```
+sudo usermod -aG docker ${USER}
+```
+then to apply the new group:
+
+```
+su - ${USER}
+```
+To confirm user is in docker group:
+
+```
+group
```

Comment:
I think the command is "groups" instead of "group"

Reply:
We can also have a pointer to the "Linux postinstall" steps for the docker ubuntu install: https://docs.docker.com/engine/install/ubuntu/